ballerina/Ballerina.toml

```
@@ -1,7 +1,12 @@
 [package]
-org = "nuvindu"
+org = "nuvindu_dias"
 name = "pipe"
 version = "0.1.0"
+authors = ["Ballerina"]
+keywords = ["pipe"]
+repository = "https://github.com/Nuvindu/module-pipe"
+license = ["Apache-2.0"]
+distribution = "2201.0.1"
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/native-0.1.0-SNAPSHOT.jar"
+path = "../native/build/libs/pipe-native-0.1.0-SNAPSHOT.jar"
```

Let's configure the IDE to add an empty line at the end of each file.

```suggestion
path = "../native/build/libs/pipe-native-0.1.0-SNAPSHOT.jar"

```